src/dura.y

```diff
@@ -4622,14 +4626,35 @@ int main(int argc, char *argv[]) {
   fname_txt = malloc(PATH_MAX);    assert(fname_txt != NULL);
   fname_no_ext = malloc(PATH_MAX); assert(fname_no_ext != NULL);
 
-  strncpy(fname_no_ext, argv[fileArg], PATH_MAX);
-  strncpy(fname_asm, fname_no_ext, PATH_MAX);
+  // Getting last file as src file (using fileArg, counter of files)
+  strncpy(fname_asm, argv[fileArg], PATH_MAX);
+
+  if (relative_path) {
+    // Reusing array to change dir, latter it will have the correct values
+    strncpy(fname_no_ext, fname_asm, PATH_MAX);
+    for (t = strlen(fname_no_ext) - 1; (fname_no_ext[t] != '/') && t; t--);
```

Comment:
esto no funcionará en Windows...

Reply:
https://github.com/Fubukimaru/asMSX/pull/134/commits/b6168e4252820605931259ed96c7a3cb032fb3b3 Will this work?